src/pages/partners/deriv-prime/index.tsx

```diff
@@ -1,13 +1,21 @@
 import React from 'react'
-import { WithIntl } from 'components/localization/withIntl'
 import Layout from 'components/layout/layout'
```

Comment:
`import Layout from 'features/components/templates/layout'` use this component..